alpha/scripts/categoryCount.php

```diff
@@ -3,11 +3,12 @@
 
 require_once(__DIR__ . '/bootstrap.php');
 
-
-if(!count($argv))
-	die("No partner_id passed to script!");
+if(count($argv)<2)
```

Comment:
@MosheMaorKaltura  why not just use $argc?

Reply:
Actually , why not?